src/generators.rs

```diff
@@ -223,18 +452,23 @@ fn helper_define_fungible_token_statement() -> (String, String) {
 }
 
 /// todo: remove function name input for the generator functions that map to a single clarity fn?
-fn gen_create_ft(_function_name: &'static str, scale: u16) -> String {
+fn gen_create_ft(_function_name: &'static str, scale: u16) -> (Option<String>, String) {
     let mut body = String::new();
 
     for _ in 0..scale {
         let (statement, _) = helper_define_fungible_token_statement();
         body.push_str(&statement);
     }
-    body
+    (None, body)
 }
 
-/// Creates a random principal to use in a clarity contract. The output includes the prefixing tick mark.
-fn helper_create_principal() -> String {
+fn helper_create_principal_in_hex() -> String {
+    let privk = Secp256k1PrivateKey::new().to_hex();
+
+    format!("0x{} ", privk)
+}
+
+fn helper_create_principal_string() -> String {
```

Comment:
is there a purpose for this helper? doesn't seem like it's used anywhere else besides `helper_create_principal()`